### DISCUSSION TITLE
DC50F16C-247D-4CDA-A68D-B0CBAAAC4A6A

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#203] [General]
# DC50F16C-247D-4CDA-A68D-B0CBAAAC4A6A

The CST-100 (Starliner) is one of two crewed US spacecraft alongside the SpaceX Dragon 2 designed to transport crew to and from the ISS and any other future commercial space station. Designed and manufactured by Boeing, this partially reusable spacecraft first flew in 2019, and has carried humans to space since 2024

https://celestia.mobi/addon?item=DC50F16C-247D-4CDA-A68D-B0CBAAAC4A6A

**Comment by dave-kaye** [comment]
This is an excellent model of the Starliner. I would request two modifications however.

First, is it possible to break the outer skin material into two separate materials? On the images I have seen, the capsule color appears blueish-grey while the lower service module is white and the connection piece between the two is blueish-grey.

Second, on OTF-2 Boeing added a docking hatch re-entry cover (Wikipedia - Starliner) similar to the SpaceX Dragon design. It is underneath the ascent cover and is less cone shaped and more flat topped. Is it possible to add this cover in the open position (docking position) as seen on the picture above?

Thank you.

**Reply by DaveBowman2001** [reply]
> First, is it possible to break the outer skin material into two separate materials?

The texture I've used for the entire spacecraft (both SM and RC) are only one image, but I can work on a slight repaint on it

> Second, on OTF-2 Boeing added a docking hatch re-entry cover (Wikipedia - Starliner) similar to the SpaceX Dragon design... Is it possible to add this cover in the open position (docking position) as seen on the picture above?

Yes, this is planned for future updates of this add-on. To minimize storage space, I plan to make the docking cover as a separate 3D model so that we could still simulate the original OFT-1 mission and all subsequent missions while using the same 3D model as before
